tests/unit/test_memory_signer.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from cytoolz import assoc
+from eth_account._utils.legacy_transactions import Transaction
+from eth_utils import to_checksum_address
+from hexbytes import HexBytes
+
+from nucypher.blockchain.eth.constants import LENGTH_ECDSA_SIGNATURE_WITH_RECOVERY
+from nucypher.blockchain.eth.signers import InMemorySigner, Signer
+from tests.unit.test_web3_signers import TRANSACTION_DICT
+
+
+@pytest.fixture(scope="function")
+def signer():
+    _signer = InMemorySigner()
+    return _signer
```

Comment:
Why not just `return InMemorySigner()`?

Reply:
It's a function of stylistic habit - I prefer block vs inline `return` usage so that a there is a line that can be used in the debugger inside the function body.